packages/core/content-manager/admin/src/preview/index.ts

```diff
@@ -12,14 +13,10 @@ const previewAdmin = {
       return;
     }
 
-    const contentManagerPluginApis = app.getPlugin('content-manager').apis;
+    const contentManagerPluginApis = app.getPlugin('content-manager')
+      .apis as ContentManagerPlugin['config']['apis'];
 
-    if (
-      'addEditViewSidePanel' in contentManagerPluginApis &&
-      typeof contentManagerPluginApis.addEditViewSidePanel === 'function'
-    ) {
-      contentManagerPluginApis.addEditViewSidePanel([PreviewSidePanel]);
-    }
+    contentManagerPluginApis.addEditViewSidePanel([PreviewSidePanel]);
```

Comment:
@Marc-Roig following your feedback on the previous PR I found this ContentManagerPlugin type I can use here